tests/vm_test.go

```diff
@@ -1786,7 +1786,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				Expect(err.Error()).To(ContainSubstring("running"))
 			})
 
-			It("[test_id:4659]should succeed", func() {
+			PIt("[test_id:4659]should succeed", func() {
```

Comment:
Can we us `Skip("special condition wasn't met")` so we can describe why we skip this ?

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/9d242e2300e180c563c8177bc157ac5a76391444..d7d8990fe3fa47ebe71f527a9e3b2dd3397b6b26)